borealis/utils/protobuf/SConscript

```diff
@@ -28,6 +28,7 @@
 Import('*')
 import os
 
-RadarProtos = ['driverpacket.proto','rxsamplesmetadata.proto']
+RadarProtos = ['driverpacket','rxsamplesmetadata']
 Proto(RadarProtos)
+Lib('messages', protos=RadarProtos)
```

Comment:
messages lib?

Reply:
This was my fix for how to keep the protobufs in /borealis/borealis/utils/protobuf (since they are shared between Python and C++, so I didn't want to bury them in usrp_drivers). By building the protobuf classes as a Library, I can just link the library and everything builds properly. I'm sure there are other ways to do this, but this is one that works.